### PR TITLE
[ADP-3285] Support sanchonet

### PR DESCRIPTION
**Pull request by paolino**
- [x] Add sanchonet configs
- [x] Update README to support sanchonet
- [x] Update docker-compose to support sanchonet
- [x] Add a script to start an rc-latest wallet and node 8.7.3 in sanchonet and ikar via docker compose
- [x] Add a script to start the wallet in the current code and node 8.7.3 in sanchonet via nix (ikar still missing)

ADP-3285

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,19 +58,21 @@ start with Daedalus.
 
 The `cardano-wallet` executable is an HTTP server that manages your wallet(s). Here is one way to start both node and wallet using Docker:
 
+THIS EXPOSES THE WALLET PORT. DO NOT USE IT WITH mainnet !
```
Comment (HeinrichApfelmus) [inline README.md]:
```suggestion
THE FOLLOWING EXPOSES THE WALLET PORT. DO NOT USE IT WITH mainnet !
```

File: configs/cardano/sanchonet/alonzo-genesis.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        "exUnitsMem": 50000000,
+        "exUnitsSteps": 40000000000
+    },
+    "maxCollateralInputs": 3,
+    "maxTxExUnits": {
+        "exUnitsMem": 10000000,
+        "exUnitsSteps": 10000000000
+    },
+    "maxValueSize": 5000
+}
```
Comment (HeinrichApfelmus) [inline configs/cardano/sanchonet/alonzo-genesis.json:188]:
```suggestion
}

```

(newline at end of file)

File: configs/cardano/sanchonet/genesis.json
Comment (HeinrichApfelmus) [inline configs/cardano/sanchonet/genesis.json:1]: Borken?

File: scripts/sanchonet.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+export WALLET_TAG=rc-latest
+export NODE_TAG=8.7.3
+export WALLET_PORT=8090
+export NETWORK=sanchonet
+export WALLET_DB=/tmp/cardano-wallet-sanchonet.db
+export NODE_DB=/tmp/cardano-node-sanchonet.db
+
+docker-compose pull
+
+docker-compose up
```
Comment (HeinrichApfelmus) [inline scripts/sanchonet.sh]:
```suggestion
docker-compose up

```

Newline at end of file

**Review by HeinrichApfelmus — APPROVED** [review]
The file `sanchonet/genesis.json` seems broken. 🤔  Good to merge after that.

File: configs/cardano/sanchonet/alonzo-genesis.json (hunk 2 above)
Comment (paolino) [reply configs/cardano/sanchonet/alonzo-genesis.json:188]: this will change the hash ?

**Pull request merged**